apps/files/models.py

```diff
@@ -17,6 +18,7 @@ class File(BaseTeamModel):
     content_type = models.CharField(blank=True)
     schema = models.JSONField(default=dict, blank=True)
     expiry_date = models.DateTimeField(null=True)
+    summary = models.TextField(max_length=400, blank=True)  # This is roughly 1 short paragraph
```

Comment:
I can see us needing to increase this limit. How about we go with 1024?

Reply:
Sure https://github.com/dimagi/open-chat-studio/pull/1259/commits/b461bfa628e68fc30a4240c2c933275d7c14a0e2